src/rgw/driver/rados/rgw_user.cc

```diff
@@ -102,6 +102,7 @@ static void dump_access_keys_info(Formatter *f, RGWUserInfo &info)
     f->dump_format("user", "%s%s%s", s.c_str(), sep, subuser);
     f->dump_string("access_key", k.id);
     f->dump_string("secret_key", k.key);
+    f->dump_bool("active", k.active);
```

Comment:
Oh hm, this is a bool?  So instead of "active" and "deactive" / "inactive" we want "true" or "false" as the values for the option?